build.gradle

```diff
@@ -582,6 +583,41 @@ configure(updateBuildScript) {
     description = 'Updates the build script to the latest version'
 }
 
+// Deobfuscation
+
+def deobf(String sourceURL, String fileName) {
+    def bon2File  = "$projectDir/deobf/BON2-2.5.0.jar"
+    def cacheFile = "$projectDir/deobf/out/" + fileName + ".jar"
+    def deobfFile = "$projectDir/deobf/out/" + fileName + "-deobf.jar"
+
+    if(file(deobfFile).exists()) {
+        return files(deobfFile)
+    }
+
+    download {
+        src 'https://github.com/glowredman/BON2/releases/download/2.5.0/BON2-2.5.0.CUSTOM-all.jar'
```

Comment:
Would you mind forking this into the GTNH repo (and add the @admin group as an admin).  Trying to keep all deps internal to the org.

Reply:
@mitchej123 https://github.com/GTNewHorizons/BON2/

Added gtnewhorizons/admin as admin and removed myself. Other settings should be done by you or @boubou19 so I don't break stuff